domain/src/main/java/com/aconno/acnsensa/domain/repository/InMemoryRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.aconno.acnsensa.domain.repository
+
+import com.aconno.acnsensa.domain.model.SensorType
+import com.aconno.acnsensa.domain.model.readings.Reading
+
+interface InMemoryRepository {
+
+    fun addReadingFor(reading: Reading)
```

Comment:
What is meaning of the "For" part of the function name?

Reply:
It's a typo